linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/dashboard.js

```diff
@@ -60,3 +69,181 @@ var Dashboard = (function() {
     };
   };
 })();
+
+var RouterController = (function () {
+  // Contains all the router components, e.g. summaries, graphs, etc
+
+  function initializeRouterContainers(selectedRouter, routers, $parentContainer) {
+    var routerData = !selectedRouter ? routers.data : { router: routers.data[selectedRouter] };
+
+    var domContainers = _.reduce(routerData, function(containers, router) {
+      var $routerEl = $("<div />").addClass("router router-" + router.router);
+      $routerEl.append($("<div />").addClass("summary"));
+      $routerEl.append($("<div />").addClass("server"));
+      containers[router.router] = $routerEl;
+      return containers;
+    }, {});
+
+    _.each(domContainers, function($container, routerName) {
+      $parentContainer.append($container);
+    });
+
+    return domContainers;
+  }
```

Comment:
this would be easier to read as a handlebars template eg
```
{{#each router}}
 <div class="router router-{{router}}">
    <div class="summary"></div>
    <div class="server"></div>
</div>
{{/each}}
```